### PR TITLE
Improve coverage

### PR DESCRIPTION
**Pull request by tsamaya**

### CODE REVIEW

File: src/test/java/de/meggsimum/w3w/What3WordsTest.java [hunk 1]
```diff
@@ -69,10 +103,29 @@ public void testPositionToWords() {
...
+			assertTrue(false);
+		}
+	}
+
+	/**
 	 * Tests the position -> words API wrapper after changing the language
 	 */
 	public void testChangeLang() {
 		What3Words w3w = new What3Words(API_KEY);
+		String defaultLanguage = w3w.getLanguage();
```
Comment (chrismayer) [inline src/test/java/de/meggsimum/w3w/What3WordsTest.java]: This line does not make any sense, does it? Can you remove it please?

File: src/test/java/de/meggsimum/w3w/What3WordsTest.java [hunk 2]
```diff
@@ -104,4 +157,31 @@ public void testWhat3WordsException() {
 		assertTrue(thrown);
 	}
 
+	public void testNonAsciiCharatersFR() {
```
Comment (chrismayer) [inline src/test/java/de/meggsimum/w3w/What3WordsTest.java:171]: Would you mind adding some little API-doc blocks, like the other test methods have?

File: src/test/java/de/meggsimum/w3w/What3WordsTest.java [hunk 3]
```diff
@@ -104,4 +157,31 @@ public void testWhat3WordsException() {
...
+			coords = w3w.wordsToPosition(words);
+			assertEquals(2, coords.length);
+			assertEquals(-21.951124, coords[0]);
+			assertEquals(166.685219, coords[1]);
+		} catch (Exception e) {
+			assertTrue(false);
+		}
+	}
+
+	public void testNonAsciiCharactersDE() {
```
Comment (chrismayer) [inline src/test/java/de/meggsimum/w3w/What3WordsTest.java:188]: Would you mind adding some little API-doc blocks, like the other test methods have?

**Comment by chrismayer** [comment]
Hi @tsamaya,

thanks for this! I really like it to have an improved test coverage. I added some minor line comments. would you mind addressing them? After that I think this is good to go.

File: src/test/java/de/meggsimum/w3w/What3WordsTest.java (hunk 1 above)
Comment (tsamaya) [inline src/test/java/de/meggsimum/w3w/What3WordsTest.java]: Hi, yes it does not have a real sens but in fact, it tests the getLanguage method. I should a real test with it.

**Comment by tsamaya** [comment]
Hi @chrismayer,
I pushed a new commit with API Docs and a test for getLanguage

**Comment by chrismayer** [comment]
Thanks for your ongoing effort on this, @tsamaya! :+1:  This is good to go now. I will merge this.

**Pull request merged**